sig-security-charter.md

```diff
@@ -79,3 +79,27 @@ SIG Security's scope does not include:
 * Recommendations or endorsements of specific commercial product vendors or
   cloud providers
 * Direct vulnerability response, this belongs to the relevant repo owners.
+
+## Membership, Roles, and Responsibilities
+
+SIG Security is open to all OpenTelemetry contributors and community members
+interested in security. We welcome all levels of experience, from those who are
+new to security to those who are seasoned experts. We encourage participation
+from a diverse range of backgrounds and perspectives, as security is a
+multifaceted field that benefits from a variety of viewpoints.
```

Comment:
This appears to be mostly a duplicate of what we already have for the community in general: https://github.com/open-telemetry/community/blob/main/mission-vision-values.md#community-values

Reply:
Yes, it makes sense to align the values because this project is part of OpenTelemetry.
I think it is reasonable to call it out here to encourage contributions, an alternative approach is to not have this statement and link to the community page, which could be less convenient for the users?

Reply:
I think a link is fine. This seems unnecessary duplication. I don't see anything specific to SIG Security in this paragraph.

Reply:
Do you have a concrete suggestion how to rephrase this? We cannot put a bare link because it will break markdownlint.

Reply:
@tigrannajaryan I'll merge for now, please keep commenting here and I can follow up in a separate PR.